src/helpers/isSearchResultSame.js

```diff
@@ -0,0 +1,27 @@
+// Currently there is no unique id with search result. To match if 2 results are same
```

Comment:
I think it's important to explain why we are doing this conversion. I was wondering the previous implementation and took me a time to figure out how and why we are doing this. I wanted to document this so that next person doesn't have to go through the same time wasting effort that I had to.

Reply:
a commit message is project history as well, but you fair point. Since we have a bunch of comments, I guess this is fine